packages/plugins/model-define/cycle-gan-model-define/CycleGAN/utils/option_utils.py

```diff
@@ -0,0 +1,71 @@
+
+import numpy as np
+from pprint import pprint
+
+class Option(object):
```

Comment:
These parameters have been defined in index.ts for model-define. This file is not required

Reply:
Done.